### PR TITLE
Add tests for DeleteEventCommand & related classes

### LINKED ISSUE #166: Add tests for DeleteEventCommand & related classes

### PR DESCRIPTION
**Pull request by vevek**
Closes #166

### CODE REVIEW

**Comment by vevek** [comment]
Ready for review, thanks!

File: src/test/java/seedu/address/logic/commands/DeleteEventCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                INDEX_FIRST, lastEventFromProject1);
+        assertEquals(deleteEventFromProject1Command, deleteEventFromProject1CommandCopy);
+
+        // different types -> returns false
+        assertNotEquals(deleteEventFromProject1Command, 1);
+
+        // null -> returns false
+        assertNotEquals(deleteEventFromProject1Command, null);
+
+        // different person -> returns false
```
Comment (samuelfangjw) [inline src/test/java/seedu/address/logic/commands/DeleteEventCommandTest.java]: typo here

**Review by samuelfangjw — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/address/logic/commands/DeleteEventCommandTest.java (hunk 1 above)
Comment (vevek) [reply src/test/java/seedu/address/logic/commands/DeleteEventCommandTest.java]: Nice. fixed!

**Review by samuelfangjw — DISMISSED** [review]

**Review by samuelfangjw — APPROVED** [review]
lgtm

**Pull request merged**